src/backend/web/handlers/team.py

```diff
@@ -119,3 +122,27 @@ def team_list(page: int) -> str:
         "current_page": page,
     }
     return render_template("team_list.html", template_values)
+
+
+@cached_public
+def avatar_list(year: Optional[Year] = None) -> Response:
+    year = year or SeasonHelper.get_current_season()
+
+    valid_years = list(range(2018, SeasonHelper.get_max_year() + 1))
+    valid_years.remove(2021)  # No avatars in 2021 :(
+
+    if year not in valid_years:
+        abort(404)
+
+    avatars_future = Media.query(Media.media_type_enum
+                                 == MediaType.AVATAR, Media.year == year).fetch_async()
```

Comment:
Could we build out a query here? (See the existing media queries for guidance https://github.com/the-blue-alliance/the-blue-alliance/blob/py3/src/backend/common/queries/media_query.py) That way we leverage our caching layer and abstract away any of the query stuff

Reply:
OH sure right. This is interesting actually. I wonder if we could bake this in to a paginated cached query (the way we still need to build out a paginated query https://github.com/the-blue-alliance/the-blue-alliance/pull/3389)

Either way - yeah Eugene is probably right here. We should build the query/caching stuff in to the method. But it does need the caching.